lib/graphql_devise.rb

```diff
@@ -7,3 +7,6 @@
 module GraphqlDevise
   class Error < StandardError; end
 end
+
+require 'graphql_devise/user_error'
```

Comment:
Any particular reason to not place these 2 on line 6

Reply:
This one in particular does not inherit from the error declared in the above line. But might be good to require all errors after our main error declaration like DTA does